gemfiles/Gemfile.public_suffix_2

```diff
@@ -0,0 +1,34 @@
+source 'https://rubygems.org'
```

Comment:
@sporkmonger It's for dev or test, just like the main /Gemfile (which isn't used when building the gem, the gemspec is) which doesn't have comments about that.